docs/release-checklist.rst

```diff
@@ -95,8 +129,10 @@ they will need to evaluate which contributors' signatures they trust.
 - (all steps above are completed)
 - sign the release
 
-  - git tag -s -u 0xE34E62D06D0E69CFCA4179FFBDE0D31D68666A7A -m "release Tahoe-LAFS-1.15.0rc0" tahoe-lafs-1.15.0rc0
-  - (replace the key-id above with your own)
+  - ``git tag -s -u 0xE34E62D06D0E69CFCA4179FFBDE0D31D68666A7A -m "release Tahoe-LAFS-1.16.0rc0" tahoe-lafs-1.16.0rc0``
+
+*Replace the key-id above with your own, which can simply be your email if's attached your fingerprint.*
```

Comment:
maybe make these additional bullet- points? (Or at least, the markup seems weird). grammar: "if it's attached".

Reply:
I think bullet points are part of the instructions this is an "aside", the markup put's it in italic so it stands out from the instructions as a TIP.

Reply:
I'm mostly concerned with how it looks like in emacs, as text.
If you really want a rendered "aside" then `.. note:` I think is the right thing?

Reply:
True.